directions/forms/forms380.py

```diff
@@ -500,22 +529,44 @@ def printForm():
 
         issledovaniya = dir.issledovaniya_set.all()
         opinion = [
-            [Paragraph('Цель и наименование исследования', styleT), Paragraph('Материал - место взятия', styleT), Paragraph('Показания   к    обследованию', styleT)],
+            [
+                Paragraph('Цель и наименование исследования', styleCenterBold),
+                Paragraph('Материал - место взятия', styleCenterBold),
+                Paragraph('Показания   к    обследованию', styleCenterBold),
```

Comment:
Почему не как то так?
```suggestion
                Paragraph('Показания к<br/>обследованию', styleCenterBold),
```
или просто
```suggestion
                Paragraph('Показания к обследованию', styleCenterBold),
```